test/unit/BLSApkRegistryUnit.t.sol

```diff
@@ -800,8 +990,8 @@ contract BLSApkRegistryUnitTests_quorumApkUpdates is BLSApkRegistryUnitTests {
         uint256 blockGap,
         uint256 randSeed
     ) external {
-        cheats.assume(numRegistrants > 0 && numRegistrants < 100);
-        cheats.assume(blockGap < 100);
+        numRegistrants = bound(numRegistrants, 1, 100);
+        blockGap = bound(blockGap, 0, 10);
```

Comment:
any reason for switching the upper bound here to 10 from 100?
seems fine either way tho

Reply:
nice catch that was a typo

Reply:
[2bd61f3](https://github.com/Layr-Labs/eigenlayer-middleware/pull/247/commits/2bd61f30ac0fef0bd91a8caedca26b1d31facdda)